src/bin/vip-dev-env-sync-sql.js

```diff
@@ -41,6 +40,12 @@ const appQuery = `
 		primaryDomain { name }
 		uniqueLabel
 		isMultisite
+		wpSites {
```

Comment:
This has pagination. So by default it won't pull all the sites.

Todo: find a way around.

Reply:
Also:
```suggestion
		wpSitesSDS {
```
Because `wpSites` will be deprecated.

Reply:
yes